liquidity/liquidity_test.go

```diff
@@ -1555,7 +1555,7 @@ func TestFeePercentage(t *testing.T) {
 			quote: &loop.LoopOutQuote{
 				SwapFee:      60,
 				PrepayAmount: 30,
-				MinerFee:     1,
+				MinerFee:     10,
```

Comment:
Since we changed 100X to 10X this now needs to be bumped for the test to return the expected error